db/migrate/20150818142457_add_vm_template_id_to_vms.rb

```diff
@@ -0,0 +1,5 @@
+class AddVmTemplateIdToVms < ActiveRecord::Migration
+  def change
+    add_column :vms, :vm_template_id, :bigint
```

Comment:
@blomquisg @Fryguy how do we make this relationship in other providers where we support this?  This feels like a one-off implementation.